ivoatexDoc.tex

```diff
@@ -810,6 +810,12 @@ \subsubsection{Local Bibliographies}
 \emph{url} points to the actual versioned landing page, not the generic
 one for the standard.
 
+Also note that while the PDF pipeline will happily let you use
```

Comment:
I worry a bit about possible confusion resulting from changing the reference from what one can discover and search for via ADS and other resources.  Is this admonition just for `\verb||` or is it recommending changing `&` to `+` in `citep{}` also?  Anecdotally I don't see rendering or referencing problems with `&` in either, so I just want to be sure that the risk of bad behavior outweighs the risk of confusion.  

Also, since I couldn't reproduce a specific problem in the rendering, I couldn't convince myself that `+` (also sometimes a special character) would be safer than `&`.

Reply:
A drive-by comment: BibTeX has a larger number of legal characters than one might expect (the syntax for it was borrowed from a formatting system called ‘Scribe’, so it's not intrinsically TeX-friendly by design), thus as far as BibTeX, and .bst files, are concerned, ampersands is a perfectly legitimate character to find in a citation key, as are a variety of what TeX regards as non-‘letter’ characters, and LaTeX's usage of BibTeX output just has to cope.  This of course does play merry hell with *ML, of course: it probably isn't worth handling these in `tth`, but if this is to remain unhandled, it should be closed as ‘tth can't cope’ rather than ‘bad characters!’.

Also, it is quite possible for `\verb|...|` to go wrong in some circumstances.  That command plays some simple catcode tricks, of course, and these do tend to break in ‘moving arguments’, which include just about anything written to a file..